aep/general/0136/aep.md.j2

```diff
@@ -49,28 +54,17 @@ apply consistently:
 
 - The `operationId` **should** be a verb followed by a noun.
   - The `operationId` **must not** contain prepositions ("for", "with", etc.).
-- The HTTP method for custom operations **should** usually be `POST`, unless
-  the custom method maps more strongly to another HTTP verb.
-  - Custom operations that serve as an alternative to get or list operations
-    (such as `Search`) **should** use `GET`, and require no request body. These
-    operations **must** be idempotent and have no state changes or side effects
-    (they should be safe as defined in [RFC 7231][]).
-  - Custom operations **should not** use `PATCH` or `DELETE`.
-- The HTTP URI **must** use a `:` character followed by the custom verb
-  (`:archive` in the above example), and the verb in the URI **must** match the
-  verb in the `operationId`.
-  - If word separation is required, `camelCase` **must** be used.
```

Comment:
Let's go back to the Google guidance here.

> - The HTTP method must be GET or POST:
>   - GET must be used for methods retrieving data or resource state.
>   - POST must be used if the method has side effects or mutates resources or data.
> - The HTTP URI must use a : character followed by the custom verb (:archive in the above example), and the verb in the URI must match the verb in the name of the RPC.
>   - If word separation is required, camelCase must be used.

Reply:
For the first bullet point, see discussion [here](https://github.com/aep-dev/aep.dev/pull/103/files#r1518530318).

Second bullet point reverted to Google style.

Reply:
@rofrankel the "here" link just brings me back to the files tab. Is there something more specific? Maybe copy/paste the key points here?

Reply:
Oh, that's annoying, sorry.  Here's a second attempt at producing the right link: https://github.com/aep-dev/aep.dev/pull/103#discussion_r1518530318 (just worked for me when I tried it in a new tab).

But also, I'll quote it here:

> The AIP guidance says it MUST be POST or GET: I think this is a pretty valuable change that was made after the original import: https://google.aip.dev/136#guidance. Can we add it?

> Discussed live:

> * Rich is concerned that making this a must without any kind of informal proof that there won't be a valid use case for e.g. DELETE risks someone running up against the AEPs not supporting a valid use case. (E.g. delete this node, reparent its children, and temporarily cache the new parent for for nodes newly parented to the deleted node.)
> * Yusuke thinks this guidance lacks context; people may just implement patch semantics with a POST request.
> * Conclusion: refer to the IETF document on HTTP methods and say to follow those. RFC 7231 (and whichever one has PATCH). Demote this guidance to a "in practice, most of the time" style note.

Reply:
Created https://github.com/aep-dev/aep.dev/issues/128 to revisit this - merging in advance of the barn raising.